pkg/resources/common/formatter.go

```diff
@@ -287,6 +290,11 @@ func convertMetadataTimestampFields(request *types.APIRequest, gvk schema2.Group
 	}
 }
 
+func isDuration(value string) (time.Duration, bool) {
+	d, err := time.ParseDuration(value)
```

Comment:
Should I use `ParseTimestampOrHumanReadableDuration` here instead?
```suggestion
	d, err := ParseTimestampOrHumanReadableDuration(value)
```